config/scheduler-chromeos.yaml

```diff
@@ -187,6 +187,14 @@ scheduler:
       result: pass
     platforms: *mediatek-platforms
 
+  - job: kselftest-discoverable-devices-probe
+    <<: *lava-job-collabora
+    event:
+      channel: node
+      name: kbuild-gcc-10-arm64-chromebook
```

Comment:
GCC was recently bumped to 12; see https://github.com/kernelci/kernelci-pipeline/commit/ac15e371e2d5cf8bd1bdc621493892346658f0e6

Reply:
Fixed.